theme/less/_content.less

```diff
@@ -18,45 +18,47 @@ article p:first-of-type img
 
 article p
 {
-  margin: 0.5em 0 0 0;
+  margin: 14px 0 0 0;
 }
 
-article p + ul
+article ul
 {
-  margin-top: 0.3em;
+  margin: 20px 0 0 0;
 }
 
-article h3 + ul
+article h1
 {
-  margin-top: 0.5em;
+  font-size: 2.4em;
+  line-height: 1.4em;
+  margin: 20px 0 0 0;
+  font-weight: 300;
+  color:#181818;
```

Comment:
We have this color also on `h2, h3, h4, h5, h6` on Drupal site.

Reply:
I have left it more grey-ish on other heading elements, because in the Drupal the font-weight is lower and having a bit of grey-ish sub headers, felt to me less distracting when comparing the Drupal and current website, it feels more "calm".

Reply:
Not really sure about this. Well, let me rely on your decision and intuition then...